src/CoreBluetooth/AdvertisementData.cs

```diff
@@ -0,0 +1,32 @@
+using System;
+using System.Collections.Generic;
+using System.Runtime.InteropServices;
+using System.Linq;
+
+using XamCore.ObjCRuntime;
+using XamCore.Foundation;
+using XamCore.CoreFoundation;
+
+namespace XamCore.CoreBluetooth {
+
+	public partial class AdvertisementData {
+		
+		public IDictionary <CBUUID, NSData> ServiceData {
```

Comment:
I think we should return a DictionaryContainer subclass here instead, maybe the user will need this dictionary somewhere else and he can pass it easily this way and get all the benefits of the strongly typed dictionary